src/Makefile

```diff
@@ -1008,9 +1013,12 @@ PROCPS_INCLUDE=-I./${EXTERNAL_PROCPS}
 
 procps_c := $(wildcard ${EXTERNAL_PROCPS}*.c)
 procps_o := $(procps_c:.c=.o)
+ifeq (${CHECK_ALPINE},YES)
+alpine_debug := "-DPATH_MAX=4096"
```

Comment:
Why is needed to include the `PATH_MAX` definition when compiling the `procps` lib. What error is throwing?

Reply:
Hi Chema, when you build Wazuh in debug mode un Alpine Linux if you don't set PATH_MAX to procps in Alpine this library cannot be built for this distribution with a message similar to this:
```
external/procps/readproc.c:1038:26: error: 'PATH_MAX' undeclared (first use in this function)
 1038 |         static char path[PATH_MAX], sbuf[1024];
      |                          ^~~~~~~~
external/procps/readproc.c:1038:26: note: each undeclared identifier is reported only once for each function it appears in
external/procps/readproc.c:1038:21: warning: unused variable 'path' [-Wunused-variable]
 1038 |         static char path[PATH_MAX], sbuf[1024];
      |                     ^~~~
external/procps/readproc.c: In function 'simple_nexttid':
external/procps/readproc.c:739:46: warning: '%s' directive output may be truncated writing up to 255 bytes into a region of size between 41 and 51 [-Wformat-truncation=]
  739 |   snprintf(path, PROCPATHLEN, "/proc/%d/task/%s", p->tgid, ent->d_name);
      |                                              ^~
external/procps/readproc.c:739:3: note: 'snprintf' output between 14 and 279 bytes into a destination of size 64
  739 |   snprintf(path, PROCPATHLEN, "/proc/%d/task/%s", p->tgid, ent->d_name);
```

Reply:
Thank you! Could you also please uppercase the variable to `ALPINE_DEBUG` just to be consistent with the other variables in the Makefile?

Reply:
Done